android/sdl_android/src/androidTest/java/com/smartdevicelink/test/rpc/notifications/OnVehicleDataTests.java

```diff
@@ -258,7 +258,7 @@ public void testJson() {
             //Set up the JSONObject to represent OnVehicleData:
 
             //TIRE_PRESSURE
-            tireStatusObj.put(TireStatus.KEY_PRESSURE_TELL_TALE, VehicleDataHelper.TIRE_PRESSURE_TELL_TALE);
```

Comment:
@kostyaBoss I would suggest reverting these changes in `OnVehicleDataTests.java`. These are valid changes, since the `pressureTellTale` functions are being renamed per the parameter, but is out of scope of the PR.